easybuild/easyconfigs/a/APBS/APBS-1.4.1-intel-2015a.eb

```diff
@@ -0,0 +1,39 @@
+name = 'APBS'
+version = '1.4.1'
+
+easyblock = 'CMakeMake'
+
+homepage = 'http://www.poissonboltzmann.org/apbs'
+description = """ APBS is a software package for modeling biomolecular solvation 
+ through solution of the Poisson-Boltzmann equation (PBE), one of the most popular 
+ continuum models for describing electrostatic interactions between molecular solutes 
+ in salty, aqueous media. """
+
+toolchain = {'name': 'intel', 'version': '2015a'}
+
+#git clone -b 1.4.1-binary-release  https://github.com/Electrostatics/apbs-pdb2pqr
+#create your own tarball
```

Comment:
Argh, what a bunch of idiots... >_<

Can we figured out which git commit ID matches 1.4.1 exactly? Maybe https://github.com/Electrostatics/apbs-pdb2pqr/commit/74fcb8676de69ed04ddab8976a8b05a6caaf4d65 ? Or maybe one of the older commits in https://github.com/Electrostatics/apbs-pdb2pqr/commits/1.4.1-binary-release ?

Then you can do something like:

```python
source_urls = ['https://github.com/Electrostatics/apbs-pdb2pqr/archive/']
sources = ['74fcb867.tar.gz']
```